docs/transaction-api.asciidoc

```diff
@@ -64,14 +64,58 @@ Arguments:
 
 * `options` - The following options are supported:
 
+** `blocked` - Blocks the associated transaction from ending till this span is ended. Blocked spans
+    automatically creates a task. Defaults to false
+
 ** `parentId` - Parent id associated with the new span. Defaults to current transaction id
 
-** `sync` - Denotes if the span is blocking (sync) or non-blocking(async). 
+** `sync` - Denotes if the span is synchronous or asynchronous.
 
 When a span is started it will measure the time until <<span-end,`span.end()`>> is called.
 
 See <<span-api,Span API>> docs for details on how to use custom spans.
 
+[float]
+[[transaction-add-task]]
+==== `transaction.addTask([taskId])`
```

Comment:
I am in favor of not documenting `addTask` and `removeTask` and instead guide users to use the `blocking` span API so we can reduce the number of concepts and also reduce confusion.

Reply:
Thoughts @jahtalab